operations/agent_monitor_test.go

```diff
@@ -52,6 +53,20 @@ func TestAgentMonitorWithJasper(t *testing.T) {
 			assert.True(t, proc.Complete(ctx))
 			assert.Zero(t, exitCode)
 		},
+		"RemoveMacOSClientOnlyDeletesFileInMacOS": func(ctx context.Context, t *testing.T, m *monitor) {
+			require.NoError(t, m.fetchClient(ctx, []string{"https://example.com"}, agentMonitorDefaultRetryOptions()))
+			fileInfo, err := os.Stat(m.clientPath)
+			require.NoError(t, err)
+			assert.NotZero(t, fileInfo.Size())
+
+			require.NoError(t, m.removeMacOSClient())
+			_, err = os.Stat(m.clientPath)
+			if runtime.GOOS == "darwin" {
```

Comment:
Slightly tangential but our MacOS variant is inactive. Should we enable it for the sake of affirming these sorts of tests?

Reply:
Good call, I activated it.